sipi/scripts/test_file_info.lua

```diff
@@ -0,0 +1,104 @@
+-- Copyright © 2015-2019 the contributors (see Contributors.md).
```

Comment:
Which spec runs this test?

Reply:
It's run by `KnoraSipiScriptsV1ITSpec`, which needs to be changed because it still uses the route `test_mediatype`. But that route does something very strange now in Sipi: it returns a PDF file containing a test image. So does any route beginning with `test` (e.g. `test_foobar`). I have no idea why. But `test_file_type` works correctly.

Reply:
Ok!